### PR TITLE
Submitting registry YAML for eBird Status and Trends Open Data on AWS.

### PR DESCRIPTION
**Pull request by tomauer**
*Issue #, if available:*

*Description of changes:*

Submitting registry YAML for eBird Status and Trends Open Data on AWS.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/ebirdst.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Name: eBird Status and Trends Model Results
+Description: |
+  The eBird Status and Trends project generates estimates of bird 
+  occurrence and abundance at a high spatiotemporal resolution. 
+  This dataset represents the primary modeled results from the
+  analysis workflow and are designed for further analysis, 
+  synthesis, visualization, and exploration.
+Documentation: https://ebird.org/science/status-and-trends
+Contact: https://help.ebird.org/customer/en/portal/emails/new
+ManagedBy: Cornell Lab of Ornithology
```
Comment (jflasher) [inline datasets/ebirdst.yaml]: Can you make this markdown like `[Cornell Lab of Ornithology](Cornell Lab of Ornithology)`

File: datasets/ebirdst.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+UpdateFrequency: Yearly
+Tags:
+  - biodiversity
```
Comment (jflasher) [inline datasets/ebirdst.yaml]: Please add `aws-pds` as a tag.

Comment (tomauer) [reply datasets/ebirdst.yaml]: Added in 7d5c6ae3ac0257bfa0f33bbe5fdc10475162f9ef

File: datasets/ebirdst.yaml (hunk 1 above)
Comment (tomauer) [reply datasets/ebirdst.yaml]: Formatted in 7d5c6ae3ac0257bfa0f33bbe5fdc10475162f9ef

**Comment by tomauer** [comment]
Requested changes made in 7d5c6ae3ac0257bfa0f33bbe5fdc10475162f9ef

**Pull request merged**